.github/dockerFiles/cluster_test_vreplication_basic/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# DO NOT MODIFY: THIS FILE IS GENERATED USING "make generate_ci_workflows"
```

Comment:
nit: can you rename the directory? dockerFiles -> docker

Reply:
Yes, Ill do that